release-please-config.json

```diff
@@ -16,7 +17,7 @@
     "packages/sdk-codegen-scripts": { "release-as": "" },
     "packages/sdk-codegen-utils": { "release-as": ""},
     "packages/sdk-node": { },
-    "packages/sdk-rtl": { "release-as": "" },
+    "packages/sdk-rtl": { "release-as": "21.0.20" },
```

Comment:
I'm sure this was a manual fix. Maybe we should rethink this behavior for the rtl package.

Reply:
sorry, forgot to annotate this. tldr I'm back to desiring to keep these as `21.0.x` till we re-name and then just let it semver? see https://github.com/looker-open-source/sdk-codegen/pull/802#discussion_r699397245